pkg/comp-functions/functions/vshnkeycloak/deploy.go

```diff
@@ -235,43 +351,21 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 			},
 		},
 		"nodeSelector": nodeSelector,
-		"metrics": map[string]any{
+		"dbchecker": map[string]any{
 			"enabled": true,
-			"serviceMonitor": map[string]any{
-				"enabled": true,
-			},
-		},
-		"httpRelativePath": comp.Spec.Parameters.Service.RelativePath,
```

Comment:
This should not be removed, as the upstream Relative path is set by default to `/auth`

Reply:
It's set here: https://github.com/vshn/appcat/pull/129/files#diff-9d57e64a014aff26a646229e449affc4afe8e96fc5238920a77758f105cddc14R368